### PR TITLE
[fix-10207] When the registered alarm plugin is empty, the error message is not clear

### PR DESCRIPTION
**Pull request by SnowMoon-Dev**
…stered alarm plugin is empty

## Brief change log

ref [#10207](https://github.com/apache/dolphinscheduler/issues/10207) , this PR only solves the problem of prompt information

## Verify this pull request

Manually verified the change by testing locally.

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java [hunk 1]
```diff
@@ -330,7 +330,7 @@ public enum Status {
 
     //plugin
     PLUGIN_NOT_A_UI_COMPONENT(110001, "query plugin error, this plugin has no UI component", "查询插件错误，此插件无UI组件"),
-    QUERY_PLUGINS_RESULT_IS_NULL(110002, "query plugins result is null", "查询插件为空"),
+    QUERY_PLUGINS_RESULT_IS_NULL(110002, "query alarm plugins result is empty, please check whether the alarm component is started and the plugin is successfully registered", "查询告警插件为空, please check whether the alarm component is started and the relevant alarm plug-in is successfully registered"),
```
Comment (SbloodyS) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java]: Did the chinese part finished?

Comment (SnowMoon-Dev) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java]: My problem, I fixed the problem of incomplete Chinese translation

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java [hunk 2]
```diff
@@ -330,7 +330,7 @@ public enum Status {
 
     //plugin
     PLUGIN_NOT_A_UI_COMPONENT(110001, "query plugin error, this plugin has no UI component", "查询插件错误，此插件无UI组件"),
-    QUERY_PLUGINS_RESULT_IS_NULL(110002, "query plugins result is null", "查询插件为空"),
+    QUERY_PLUGINS_RESULT_IS_NULL(110002, "query alarm plugins result is empty, please check whether the alarm component is started and the plugin is successfully registered", "查询告警插件为空, 请检查告警组件是否启动相关告警插件是否注册成功"),
```
Comment (SbloodyS) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java]: Is it better to use ```请检查告警插件是否启动成功以及相关告警插件是否注册成功```?

Comment (SnowMoon-Dev) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java]: yes, I optimized it according to your suggestion

Comment (SnowMoon-Dev) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java]: I think the document has also been updated, I will modify the relevant content of the document later

**Review by SbloodyS — APPROVED** [review]
LGTM

**Pull request merged**